testground/benchmark/benchmark/sendtx.py

```diff
@@ -1,13 +1,15 @@
-import time
+import queue
+import threading
 from concurrent.futures import ThreadPoolExecutor, as_completed
 
 import web3
 from eth_account import Account
 
-from .utils import export_eth_account, send_transaction, send_transactions
+from .utils import export_eth_account, send_transactions, sign_transaction
 
 TEST_AMOUNT = 1000000000000000000
 GAS_PRICE = 1000000000
+TX_SENDING_WORKERS = 1000
```

Comment:
not sure if need make it configurable based on machine

Reply:
let's figure out where the bottleneck is first.

Reply:
seems it brings an extra queue running tasks from the original ThreadPoolExecutor, which may cause overhead